### ISSUE TITLE
Brizy updates break form integrations - again...

### ISSUE THREAD

**Issue by TheCrucible** [ThemeFuse/Brizy#1331]
Every update of Brizy seems to RESET the contact and landing page forms I've created to send leads to clients. After an update, when I go in and check the submit button setup, nothing is configured as it was prior to update. This is a BIG problem for anyone serious about generating leads on a website. This is not a new issue, can we move it up on priority list please? Latest update broke 3 landing pages on a client site...

**Comment by MaxZiebell** [comment]
Keep in mind that Brizy is still in "beta". I agree totally that there are some major issues and missing functions to Brizy (archive pages etc.).

I love the interface but keep having to revert to Elementor for Client projects as I hit the limits of Brizy in Beta. Often enough (Landing pages etc.) it's totally fine and I hope it will deliver in the future on more complex setups (like you can do with Elementor, JetEngine and ACF).

**Comment by NataliaBB** [comment]
Hello @TheCrucible 
Sorry for the inconvenience! We have set this issue as #1 task for our workflow.
Will keep posting here any update.

Regards, Natalia

**Comment by DimiBaitanciuc** [comment]
We've looked into it and try to reproduce this and we can't understand how this can happen. Can you please share with us a couple of things :

- login credentials for your WP admin (if possible) this will help a lot 
- info on what version of PRO you were before the update. I presume you've updated to the latest current version (0.0.25)

(don't share the login credentials here, submit a ticket here: https://support.brizy.io/hc/en-us/requests/new)

**Comment by TheCrucible** [comment]
> We've looked into it and try to reproduce this and we can't understand how this can happen. Can you please share with us a couple of things :
> 
> * login credentials for your WP admin (if possible) this will help a lot
> * info on what version of PRO you were before the update. I presume you've updated to the latest current version (0.0.25)
> 
> (don't share the login credentials here, submit a ticket here: https://support.brizy.io/hc/en-us/requests/new or on email to [hello@brizy.io](mailto:hello@brizy.io))

================

Update: I've just updated a client site, screen captured the whole process, and sure enough, the Brizy form integrations reset!! Check with Dimi, I just emailed him a Youtube link showing proof it happens.

**Comment by DimiBaitanciuc** [comment]
Got it, I've passed the video to the team, they are looking into it I'll keep you in the loop. 

(just a small clarification, that I've included in the email as well. The video helps not as "proof" because we believed you in the first place. The video (hopefully) will help us understand what we need to fix by understanding what the pre conditions of your particular case are. Not a big deal, but I wanted to clear that out. I'll reply on email.)

Cheers.
